runtime/lua/vim/lsp/util.lua

```diff
@@ -88,6 +88,10 @@ function M.apply_text_edits(text_edits, bufnr)
   -- Reverse sort the orders so we can apply them without interfering with
   -- eachother. Also add i as a sort key to mimic a stable sort.
   table.sort(cleaned, edit_sort_key)
+  local current_bufnr = api.nvim_get_current_buf()
+  if not api.nvim_buf_is_loaded(bufnr) then
+    api.nvim_set_current_buf(bufnr)
```

Comment:
you don't need to use `nvim_set_current_buf` to load a buffer, just use `vim.fn.bufload(bufnr)`

Reply:
Thank you for the pointer. I pushed a fixup.